recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch

```diff
@@ -19,7 +19,7 @@ index af8099a407..8f73453a6c 100644
 +            # relocatable:
 +            # https://developercommunity.visualstudio.com/comments/623156/view.html
 +            d1trimfile_opts = []
-+            if 'SRC_DIR' in os.environ and os.path.basename(self.cc) == "cl.exe":
++            if 'SRC_DIR' in os.environ:
```

Comment:
Why?

Reply:
This is the same patch from your PR https://github.com/conda-forge/setuptools-feedstock/pull/189.